storybook/stories.gen.js

```diff
@@ -1,15 +1,15 @@
 // DO NOT EDIT: Auto-generate this file by running `yarn generate`
-import '../src/components/hv-date-field/stories/index.js';
-import '../src/components/hv-image/stories/index.js';
-import '../src/components/hv-list/stories/index.js';
-import '../src/components/hv-option/stories/index.js';
-import '../src/components/hv-picker-field/stories/index.js';
-import '../src/components/hv-section-list/stories/index.js';
-import '../src/components/hv-select-multiple/stories/index.js';
-import '../src/components/hv-select-single/stories/index.js';
-import '../src/components/hv-spinner/stories/index.js';
-import '../src/components/hv-switch/stories/index.js';
-import '../src/components/hv-text-field/stories/index.js';
-import '../src/components/hv-text/stories/index.js';
-import '../src/components/hv-view/stories/index.js';
-import '../src/components/hv-web-view/stories/index.js';
+import '../src/components/hv-date-field/stories/index.tsx';
+import '../src/components/hv-image/stories/index.tsx';
+import '../src/components/hv-list/stories/index.tsx';
+import '../src/components/hv-option/stories/index.tsx';
+import '../src/components/hv-picker-field/stories/index.tsx';
+import '../src/components/hv-section-list/stories/index.tsx';
+import '../src/components/hv-select-multiple/stories/index.tsx';
+import '../src/components/hv-select-single/stories/index.tsx';
+import '../src/components/hv-spinner/stories/index.tsx';
+import '../src/components/hv-switch/stories/index.tsx';
+import '../src/components/hv-text-field/stories/index.tsx';
+import '../src/components/hv-text/stories/index.tsx';
+import '../src/components/hv-view/stories/index.tsx';
+import '../src/components/hv-web-view/stories/index.tsx';
```

Comment:
Do these all require extensions?

Reply:
might not - in fact index could even be omitted. Can be addressed separately